src/main/java/io/coala/jetbrains/CodeAnalysisAction.java

```diff
@@ -16,6 +19,19 @@ public void update(@NotNull AnActionEvent e) {
 
     @Override
     public void actionPerformed(AnActionEvent e) {
-        // TODO: insert action logic here
+        final Project project = e.getProject();
+
+        if(project == null || project.isDisposed()) {
+            return;
+        }
+
+        final ProjectSettings projectSettings = project.getComponent(ProjectSettings.class);
+        final CodeAnalysisExecutor executor = project.getComponent(CodeAnalysisExecutor.class);
+        final CodeAnalysisTask codeAnalysisTask = new CodeAnalysisTask(project);
+
+        projectSettings.setExecutable("/path/to/coala");
+        projectSettings.addSectionToFilter("jetbrains");
```

Comment:
Needs to be a configurable option as well.

Reply:
Yes it is.